PlayCover/Utils/DataExtensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DataExtensions.swift
+//  PlayCover
+//
+
+import Foundation
+
+extension String {
```

Comment:
Theres a String extension in the URLExtensions file, which you might want to move here, or move this definition there.

Reply:
The String extension in the URLExtensions file is relevant to URLs